Tasks/Maven/task.json

```diff
@@ -359,6 +384,7 @@
         },
         {
             "name": "checkstyleAnalysisEnabled",
+            "aliases": [ "checkstyleRunAnalysis" ],
```

Comment:
Following the pattern for sonarQube booleans above, you can also go with checkStyleEnableRunAnalysis